.github/workflows/pull-request.yml

```diff
@@ -8,12 +8,12 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - name: Checkout
-              uses: actions/checkout@v2
+              uses: actions/checkout@v4
 
             - name: NPM install
-              uses: actions/setup-node@v1
+              uses: actions/setup-node@v3
               with:
-                  node-version: 12.x
+                  node-version: 16.x
```

Comment:
We should replace this entire file with this [reusable workflow](https://github.com/mparticle-integrations/mparticle-javascript-integration-heap/blob/5be0fa8ce07c227497e8f862280e070848ec1b99/.github/workflows/reusable-workflows.yml) file, or at the very least, the reusable PR workflow.

Reply:
i was thinking that.  but i also saw the lerna step and i didn't want to mess with that.  we can discuss on our zoom shortly